Core/include/Acts/Geometry/ProtoLayerHelper.hpp

```diff
@@ -33,12 +33,11 @@ class ProtoLayerHelper {
   ///
   /// @param cfg Explicit config struct
   /// @param logger logging instance
-  ProtoLayerHelper(const Config& cfg,
+  ProtoLayerHelper(const Config&,
```

Comment:
Why is there a config object to begin with if it's not used? Any reason behind this (@asalzburger maybe?)